chain/chunks/src/lib.rs

```diff
@@ -449,13 +449,29 @@ impl ShardsManager {
             || self.runtime_adapter.will_care_about_shard(account_id, parent_hash, shard_id, is_me)
     }
 
+    /// Send requests for a chunk
+    /// Depending on whether the node tracks the shard, it either downloads the full chunk
+    /// or only parts that it concerns, specifically, the part that should be owned by the node if
```

Comment:
```suggestion
    /// or only parts that it cares about, specifically, the part that should be owned by the node if
```

Reply:
we should be consistent with naming: here you mention 'concerns'/'cares' - and below you say 'tracks'. I guess they all refer to the same thing, right?